### PR TITLE
CMSCH #99 Html meta tags

### PR DESCRIPTION
**Pull request by DannySS123**
If needed, we can change the logo later.
#99

### CODE REVIEW

File: src/main/client/src/components/@pages/AchievementPage.tsx [hunk 1]
```diff
@@ -122,6 +123,7 @@ export const AchievementPage: React.FC<AchievementPageProps> = (props) => {
 
   return (
     <Page {...props} loginRequired>
+      <Helmet title="TODO" />
```
Comment (triszt4n) [inline src/main/client/src/components/@pages/AchievementPage.tsx:126]: TODO

**Comment by triszt4n** [comment]
This PR can rest, as we haven't got the REAL icons yet

**Comment by berenteb** [comment]
Please mark as draft, merging will occur when icons are available.

**Comment by berenteb** [comment]
Please do #103 wiht this

**Pull request merged**